mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java

```diff
@@ -106,20 +137,20 @@ public void updateSymbol(MilStdSymbol symbol) {
             this.sSymbolCode = symbol.getSymbolCode();
         }
 
-        SparseArray oMod = this.oRenderer.getUnitModifiers(this.getMapInstance(), this.getFeature());
-        SparseArray oAttr = this.oRenderer.getAttributes(this.getMapInstance(), this.getFeature(), isSelected());
-
-        if (!this.oModifiers.toString().equals(oMod.toString())) {
-            // The modifiers have changed so we mark it dirty.
-            this.setDirty(true);
-            this.oModifiers = oMod;
-        }
-
-        if (!this.oAttributes.toString().equals(oAttr.toString())) {
-            // The attributes has changed, mark dirty.
-            this.setDirty(true);
-            this.oAttributes = oAttr;
-        }
+//        SparseArray oMod = this.oRenderer.getUnitModifiers(this.getMapInstance(), this.getFeature());
```

Comment:
Comment as to why this is being commented out

Reply:
good catch, that was a test that shouldn't be commented